keps/sig-auth/4633-anonymous-auth-configurable-endpoints/kep.yaml

```diff
@@ -0,0 +1,39 @@
+title: Only allow anonymous auth for health endpoints
+kep-number: 4633
+authors:
+  - "@vinayakankugoyal"
+owning-sig: sig-auth
+participating-sigs:
+  - sig-auth
+status: provisional
```

Comment:
this could be implementable as far as I'm concerned, but will let @enj review before updating this

Reply:
Met with @liggitt and @enj and they agreed we can mark this as implementable after I update the kep with the AIs from comment https://github.com/kubernetes/enhancements/pull/4634#issuecomment-2153064115. 

Updated to implementable.